lib/vsc/utils/py2vs3/py3.py

```diff
@@ -36,12 +36,7 @@
 from tempfile import TemporaryDirectory  # noqa
 from urllib.parse import urlencode, unquote  # noqa
 from urllib.request import HTTPError, HTTPSHandler, Request, build_opener, urlopen  # noqa
-try:
-    # py 3.10+
-    from collections import Mapping  # noqa
-except ImportError:
-    # < py 3.10
-    from collections.abc import Mapping  # noqa
+from collections.abc import Mapping # noqa
```

Comment:
so the comment was wrong?

Reply:
it seems so. I can't get it to break. Or it was fixed upstream.

```
[wdpypere@lancelot ~]$ python3.6
Python 3.6.15 (default, Aug  2 2023, 00:00:00) 
[GCC 13.2.1 20230728 (Red Hat 13.2.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from collections.abc import Mapping
>>> 

[wdpypere@lancelot ~]$ python3.9
Python 3.9.18 (main, Aug 28 2023, 00:00:00) 
[GCC 13.2.1 20230728 (Red Hat 13.2.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from collections.abc import Mapping
>>> 

[wdpypere@lancelot ~]$ python3.12
Python 3.12.0 (main, Oct  2 2023, 00:00:00) [GCC 13.2.1 20230918 (Red Hat 13.2.1-3)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from collections.abc import Mapping
>>> 
```

Reply:
https://docs.python.org/3/library/collections.abc.html#collections.abc.Mapping says "New in version 3.5"